x-pack/plugins/uptime/public/components/queries/monitor_status_bar/monitor_status_bar.tsx

```diff
@@ -99,19 +98,11 @@ export const MonitorStatusBar = ({
               />
             </EuiFlexItem>
             <EuiFlexItem grow={false}>
-              <FormattedMessage
-                id="xpack.uptime.monitorStatusBar.healthStatus.hostMessage"
-                values={{ host }}
-                defaultMessage="Host: {host}"
-              />
-            </EuiFlexItem>
-            <EuiFlexItem grow={false}>
-              <FormattedMessage
-                id="xpack.uptime.monitorStatusBar.healthStatus.portMessage"
-                values={{ port }}
-                defaultMessage="Port: {port}"
-              />
+              <EuiLink href={full} target="_blank">
+                {full}
+              </EuiLink>
             </EuiFlexItem>
+            <EuiFlexItem grow={false}>IP: {ip}</EuiFlexItem>
```

Comment:
```suggestion
            <EuiFlexItem grow={false}>
              <FormattedMessage
                id="xpack.uptime.monitorStatusBar.healthStatus.ipAddressLabel"
                defaultMessage="IP {ip}"
                values={{ ip }}
              />
            </EuiFlexItem>
```

We have to translate bare copy.

Reply:
WDYT about just deleting the bare copy?